src/documentation/tooling/evaluation.md

```diff
@@ -0,0 +1,292 @@
+
+### Different representations
+
+Only Sphinx can comfortably handle a large number of different representations of documentations,
+such as LaTeX, EPub3, man pages, and more. The common PDF format has to be produced using an extension.
+mdBook uses a generic approach by providing a print button which compiles a single page view of the documentation.
+This can then be used to produce PDFs or other outputs via the browser's print dialogue.
+Asciidoctor has an external PDF converter, which, however, requires Ruby. It can produce manpages natively.
+Producing different presentations from Jekyll is cumbersome. PDFs can be produced via HTML to PDF conversion software, and a wrapper plugin for Jekyll exists.
+mkDocs does not support PDF conversion natively, although a plugin exists, which, however, relies on different other software which have to be installed separately.
+
+### Conclusion
+
+To calculate a final score for each tool, we have calculated the mean for all usability sub-categories in (3), and
+have calculated the mean across the three values. The results are presented in the table below, and
+are also available as a [Jupyter notebook](https://nbviewer.jupyter.org/github/sdruskat/hexatomic.github.io/blob/src/src/static/ipynb/Hexatomic%20Project%20-%20Documentation%20Tooling%20Evaluation.ipynb).
+
+|     Tool     | 1 | 3a | 3b | 3c | 3d | 3e | 3f | x̄(3) | 4 | final score |
+|--------------|---|----|----|----|----|----|----|-------|---|-------------|
+| Sphinx (rST) | 5 |  3 |  1 |  3 |  4 |  3 |  4 |   3.0 | 4 |         4.0 |
+| mdBook       | 4 |  5 |  1 |  3 |  4 |  4 |  5 |  3.67 | 3 |        3.56 |
+| Sphinx (CM)  | 3 |  5 |  1 |  3 |  2 |  3 |  4 |   3.0 | 4 |        3.33 |
+| Asciidoctor  | 3 |  4 |  1 |  4 |  3 |  3 |  3 |   3.0 | 3 |         3.0 |
+| Jekyll       | 4 |  5 |  1 |  3 |  4 |  2 |  2 |  2.83 | 2 |        2.94 |
+| mkDocs       | 3 |  5 |  1 |  3 |  4 |  3 |  2 |   3.0 | 1 |        2.33 |
+
+Due to the restrictions in usability (and slightly decreased human-readability) that reStructuredText represents (cf. section [Human-readable source format](#human-readable-source-format)),
+as well as a personal preference for markdown, we have decided to use **mdBook** for the text-based documentation of Hexatomic.
+
+#### Implementation
+
+We use local installations of mdBook on development machines to write the user, the developer & maintainer, and this project documentation.
+We use the Travis CI continuous integration platform to produce the documentation representations, and deploy them
+to GitHub Pages.
+
+The sources for the project website reside in a dedicated repository, [github.com/hexatomic/hexatomic.github.io](https://github.com/hexatomic/hexatomic.github.io).
+The sources for the Hexatomic software are held in the development repository for Hexatomic, [github.com/hexatomic/hexatomic](https://github.com/hexatomic/hexatomic).
+
+---
+
+
+[1] S. Druskat, 'A proposal for the measurement and documentation of research 
```

Comment:
This footnote does not work properly
```suggestion
[^Druskat2016]: S. Druskat, 'A proposal for the measurement and documentation of research 
```

Reply:
Fixed in f27225ac49ce782f42bc25442d4ba3cc46bf9c40.